Sources/Clappr_iOS/Classes/Plugin/Core/JumpCorePlugin.swift

```diff
@@ -1,22 +1,22 @@
 import UIKit
 
-public class DoubleTapCorePlugin: DoubleTapPlugin {
+public class JumpCorePlugin: JumpPlugin {
     
     override open var pluginName: String {
-        return "DoubleTapCorePlugin"
+        return "JumpCorePlugin"
     }
     
-    override func removeDoubleTapGesture() {
-        core?.view.removeGestureRecognizer(doubleTapGesture)
+    override func removeJumpGesture() {
```

Comment:
What do you think about renaming these methods to `addGesture` and `removeGesture`? we think they are descriptive enough :)

Reply:
Done.